### DISCUSSION TITLE
阿里云盘如何刷新缓存？

### DISCUSSION THREAD

**Discussion by hyy34** [AlistGo/alist#743] [Q&A]
也就是我在官网中新上传了个文件或者删除了个文件，如何在前端的网页中同步？

**Comment by xhofe** [comment]
https://alist-doc.nn.ci/docs/common-question#%E4%B8%8A%E4%BC%A0%E7%9A%84%E6%96%87%E4%BB%B6%E4%B8%8D%E6%98%BE%E7%A4%BA%E5%88%A0%E9%99%A4%E7%9A%84%E6%96%87%E4%BB%B6%E8%BF%98%E5%9C%A8%E4%BF%AE%E6%94%B9%E6%A0%B9%E7%9B%AE%E5%BD%95%E4%B8%8D%E7%94%9F%E6%95%88

**Comment by hyy34** [comment]
了解了，感谢。。。@@
